### ISSUE TITLE
Are you planning to release the test code as well?

### ISSUE THREAD

**Issue by pearlmary** [luca-medeiros/lightning-sam#9]
I used train.py for instance segmentation by changing to instance. It trained for batch size 1 without errors. But when I loaded the trained checkpoints (after 16th epoch), using the following command, it did not work. **### (It resulted in empty masks i.e. [] ).** 
**Am I doing anything wrong in this? Please correct me.**

image = cv2.imread('/workspace/lightning-sam/dataset/instance_version/val/vid_000002_frame0000018.jpg')
image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)

import sys
sys.path.append("..")
from segment_anything import sam_model_registry, SamAutomaticMaskGenerator, SamPredictor
sam_checkpoint = "/workspace/lightning-sam/src/out/training/epoch-000016-f10.81-ckpt.pth"
model_type = "vit_h"
device = "cuda"
sam = sam_model_registry[model_type](checkpoint=sam_checkpoint)
sam.to(device=device)

mask_generator = SamAutomaticMaskGenerator(sam)

masks = mask_generator.generate(image)

**Comment by luca-medeiros** [comment]
As mentioned on the README it currently only supports bounding box input prompts -> SamAutomaticMaskGenerator uses point prompts.

You may try loading the fine-tuned model into SamPredictor and use bbox prompts. SOmething like what I did here:
https://github.com/luca-medeiros/lightning-sam/blob/80ac78866afa8ffc2f78ec2c314258b100589df0/lightning_sam/utils.py#L52

**Comment by pearlmary** [comment]
Thank you so much. Now I understood. 
When I run utils.py, I could get the results on the images (both for original and finetuned models) when we provide the bounding box prompts. 

Similarly, can you please provide the scripts for finding the mean IOU and mean F1 scores.

Many Thanks ..

**Comment by luca-medeiros** [comment]
@pearlmary 
You can find the validation code here
https://github.com/luca-medeiros/lightning-sam/blob/80ac78866afa8ffc2f78ec2c314258b100589df0/lightning_sam/train.py#L23-L43
